docs/src/examples/components/Tree/Types/TreeExampleExperiment.shorthand.tsx

```diff
@@ -0,0 +1,166 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Tree as StardustTree } from '@stardust-ui/react'
+
+type TreeProps = {
+  items?: Object[]
+  title?: string
+  level?: number
+  size?: number
+  id?: string
+}
+
+type TreeState = {
+  open?: boolean
+}
+
+class Tree extends React.Component<TreeProps, TreeState> {
```

Comment:
nit: we started to use more and more `React.FunctionComponent`s and hooks, you can consider implementing this with hooks